model/src/main/kotlin/OrtResult.kt

```diff
@@ -246,18 +246,25 @@ data class OrtResult(
     /**
      * Return a map of all de-duplicated [Issue]s associated by [Identifier]. If [omitExcluded] is set to true, excluded
      * issues are omitted from the result. If [omitResolved] is set to true, resolved issues are omitted from the
+     * result. if [minSeverity] is not null, issues with [Issue.severity] below [minSeverity] are omitted from the
      * result.
      */
     @JsonIgnore
-    fun getIssues(omitExcluded: Boolean = false, omitResolved: Boolean = false): Map<Identifier, Set<Issue>> {
+    fun getIssues(
+        omitExcluded: Boolean = false,
+        omitResolved: Boolean = false,
+        minSeverity: Severity? = null
```

Comment:
In order to not have to deal with nullability, maybe use `Severity.entries.min()` as the default argument?

Note that this only makes sense if there is no special code that skips filtering by severity like there is at

https://github.com/oss-review-toolkit/ort/blob/968f956962301538b5e99c82a58f2da2a8cd72d9/model/src/main/kotlin/OrtResult.kt#L439-L445